project.clj

```diff
@@ -1,14 +1,17 @@
-(defproject dire "0.5.4"
+(defproject org.cyverse/dire "0.5.5-SNAPSHOT"
   :description "Erlang-style supervisor error handling for Clojure"
   :url "https://github.com/MichaelDrogalis/dire"
```

Comment:
Should we update the `url` too?

Reply:
I debated whether to update the URL or not. At the time, I left it in place because I wanted to ensure that the original author got credit. We've got the link in the README, though, so updating the URL here won't hurt; I'll update it.